site/src/components/message.md

```diff
@@ -138,3 +139,14 @@ Use the `message-sender` class to indicate message sent by the user.
 	</div>
 </article>
 ```
+
+## Loading message
+
+```html
+<div class="loading-message">
```

Comment:
Plese remove all the new CSS, you can achieve the same thing with a few atomics:

```html
<div class="border border-radius overflow-hidden">
    <div class="message message-sm">
        <div class="message-content border-none">
            <p>Working on it ...</p>
        </div>
    </div>
    <progress class="progress-bar border-radius-none" max="100"></progress>
</div>
```

But honestly, I don't really think you need to use the `message` component for this at all. You're not replacing the text within in after content is loaded, the whole new thing is rendered so I would do this instead

```html
<div class="border border-radius overflow-hidden font-size-sm">
    <p class="padding-xs">Working on it ...</p>
    <progress class="progress-bar border-radius-none" max="100"></progress>
</div>
```

Soooo I'd say either close this PR completely, or create a pattern page called "Loading" and put it there.